website/frontend/templates/docs/development.html

```diff
@@ -58,12 +58,12 @@ <h1 id="development">Development</h1>
 
       <p>
         For Mac there is <a href="/docs/build-osx/">Building Picard on OS X</a> and
-        also <a href="http://build.oxygene.sk/job/package-picard-osx-daily/" rel="nofollow">daily builds for OS X</a>.
+        also <a href="https://build.oxygene.sk/job/package-picard-osx-daily/" rel="nofollow">daily builds for OS X</a>.
```

Comment:
@samj1912 all links tested work with `HTTPS` but both of {`https`,`http}://build.oxygene.sk/job/package-picard-osx-daily/ returns `502 Bad Gateway`

Reply:
That's fine. I'll let luks know that it's down. The url is correct. Let it be https only.

Reply:
Ok. :)